### ISSUE TITLE
f flag error

### ISSUE THREAD

**Issue by SerhatTeker** [aykutkardas/status-code-check#1]
---
name: f flag issue
about: Improving f flag
---

**Describe the bug**

`-f` flag throws error.

**To Reproduce**

1. Create a file named "urls.txt" in the project root
2. Add url list to file
3. Run `go run main.go -f="./urls.txt"`
4. See the error

**Expected behavior**

Get the [Output](https://github.com/aykutkardas/status-code-check/blob/master/README.md#output-1)

**Desktop (please complete the following information):**

- OS: Linux
- Golang Version: go1.13.5 linux/amd64

**Additional context**

Error Output:

```
 200   https://jsonplaceholder.typicode.com/posts
 200   https://jsonplaceholder.typicode.com/comments
 200   https://jsonplaceholder.typicode.com/albums
 200   https://jsonplaceholder.typicode.com/photos
 200   https://jsonplaceholder.typicode.com/todos
 200   https://jsonplaceholder.typicode.com/users
Get : unsupported protocol scheme ""
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x646939]

goroutine 1 [running]:
main.getStatusCode(0xc0000c60dc, 0x0)
        $GOPATH/tmp/status-code-check/main.go:37 +0xb9
main.main()
        $GOPATH/tmp/status-code-check/main.go:63 +0x194
exit status 2
```

**Comment by SerhatTeker** [comment]
Hi. Just wanna let you know. Like go.

**Comment by aykutkardas** [comment]
The problem is that the function does not check if the entries are urls. Thank you very much for reporting this to me.

Fixed: 24a86de28ea97ae009deff4cfe2901db3f792824
